README.md

```diff
@@ -182,7 +182,7 @@ CREATE TABLE rocketmq_source (
 ) WITH (
   'connector' = 'rocketmq',
   'topic' = 'user_behavior',
```

Comment:
How about adding a _namespace_ parameter?

Reply:
If it is added in the table, then the user can add it by himself, but did not see the use of rocketmq-flink

Reply:
> If it is added in the table, then the user can add it by himself, but did not see the use of rocketmq-flink

ok, leave it in the next PR